task/import.go

```diff
@@ -47,18 +49,39 @@ func TaskImport(tctx *TaskContext) (*data.TaskOutput, error) {
 		metadataFilePath, err = saveMetadataFile(egCtx, osSess, playbackID, metadata)
 		return err
 	})
+	// Save source file to our storage
 	eg.Go(func() (err error) {
-		fullPath := videoFileName(playbackID)
+		fullPath = videoFileName(playbackID)
 		videoFilePath, err = osSess.SaveData(egCtx, fullPath, secondaryReader, nil, fileUploadTimeout)
 		if err != nil {
 			return fmt.Errorf("error uploading file=%q to object store: %w", fullPath, err)
+		} else {
+			glog.Infof("Saved file=%s to url=%s", fullPath, videoFilePath)
 		}
 		return nil
 	})
+	// Wait for async goroutines finish to run prepare
 	if err := eg.Wait(); err != nil {
-		// TODO: Delete the uploaded file
+		// TODO: Delete the source file
 		return nil, err
 	}
+	// Download our imported output file
+	fileInfoReader, err := osSess.ReadData(ctx, fullPath)
+	if err != nil {
+		return nil, fmt.Errorf("error reading imported file from output OS path=%s err=%w", fullPath, err)
+	}
+	importedFile, err := readFile(fileInfoReader)
+	if err != nil {
+		return nil, err
+	}
+	fileInfoReader.Body.Close()
+	defer importedFile.Close()
+	// RecordStream on output file for HLS playback
+	playbackRecordingId, err := RecordStream(ctx, tctx.lapi, metadata.AssetSpec, importedFile)
+	if err != nil {
+		glog.Errorf("error preparing imported file err=%w", err)
+	}
+	metadata.AssetSpec.PlaybackRecordingID = playbackRecordingId
```

Comment:
Now that the error is only logged, we need to:
 - fix formatting of error and include contextual information in the log
 - ignore the result if the error is not nil
```suggestion
		glog.Errorf("Error preparing imported file assetId=%s err=%q", tctx.OutputAsset.ID, err)
	} else {
		metadata.AssetSpec.PlaybackRecordingID = playbackRecordingId
	}
```